esp/src/eclwatch/WUDetailsWidget.js

```diff
@@ -569,8 +571,8 @@ define([
             registry.byId(this.id + "Recover").set("disabled", isArchived || !this.wu.isComplete() || this.wu.isDeleted());
             registry.byId(this.id + "Publish").set("disabled", isArchived || !this.wu.isComplete() || this.wu.isDeleted());
             registry.byId(this.id + "ZapReport").set("disabled", this.wu.isDeleted());
-            registry.byId(this.id + "Reschedule").set("disabled", !this.wu.isComplete() || this.wu.isFailed());
-            registry.byId(this.id + "Deschedule").set("disabled", this.wu.isComplete() || this.wu.isFailed());
+            registry.byId(this.id + "Reschedule").set("disabled", !this.wu.isAbleToDeschedule());
```

Comment:
I am using the opposite of what each should have enabled.

Reschedule ---> you can de-schedule
Deschedule ---> you can re-schedule